plugins/inputs/ldap_response/ldap_response.go

```diff
@@ -0,0 +1,209 @@
+package ldap_response
+
+import (
+	ctls "crypto/tls"
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"gopkg.in/ldap.v2"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type Ldap struct {
```

Comment:
Can we use `LDAP` here? This style looks quite strange to me... :-) @p-zak will the linter shout at us if we use `LDAP`?